src/index.js

```diff
@@ -27,6 +27,7 @@ const pong = require('./modules/pong.command');
 const status = require('./modules/status.command');
 const help = require('./modules/help.command');
 const eth = require('./modules/eth.command');
+const emotion = require('./modules/emotion.command');
```

Comment:
its not "command" u dont use command executor and its like a emotion/reaction scope... like a "socialization functions"

Reply:
rename to `emotion.social`. Its good or not?